src/widgets/about/about.data.tsx

```diff
@@ -66,7 +68,15 @@ const javaScriptEN: () => AboutInfo[] = () => {
     {
       id: 2,
       title: 'Worldwide mentors and trainers',
-      info: 'The Mentors and trainers of our school are front-end and javascript developers from different companies/countries. How to become a <a href="/courses#mentors-wanted">mentor</a>?',
+      info: (
```

Comment:
We're trying to get rid of all the `*.data.tsx` files, and in this case you are creating a new one. It looks like a hack. Obviously, in the future we will have to change this code back.

Reply:
I think it's best possible solution with current data structure, definitely better then using 'dangerouslySetInnerHtml'. And the only way how we can use custom components from data. 
Anyway changing data structure in future will require total rewrite of current components logic.

Reply:
I think that the logic of `*.data` files should be changed when refactoring widgets. Shared components are the simplest, they should not depend on the type of `*.data` files. I suggest leaving this file unchanged and returning to it when refactoring the widget `About`

Reply:
Without this changes task for links standardization is not fully implemented and no one will care later that custom links should be used here.

Reply:
We shouldn't use `*.tsx` components in `*.data` files. Now we're discussing the format in which we will store our `*.data` files - it could be `.json` or `.md`, which we will then parse, but it is definitely not `.tsx`

Reply:
@ansivgit the pull request is not about the data refactor. I recommend to leave this as it is with components inside and refactor this as a separate milestone after we fully migrate to FSD

Reply:
@Quiddlee I'm not suggesting refactoring the data, I'm suggesting not changing it at all, instead of creating entities that we'll definitely get rid of

Reply:
if we leave it as you suggest we'll have inconsistent links across the site

<img width="409" alt="Screenshot 2024-09-11 at 12 17 54" src="https://github.com/user-attachments/assets/3f5ffba2-6e13-4c6f-b041-3537f796d7c1">

Reply:
I am deeply convinced that increasing quantity of "bad" files is wrong and if we can't improve it now, then we should leave everything as is until the `.data` refactoring. We can set styles for the `a` tag and everything will look the same - this is not a problem at all. @SpaNb4 , what do you think?

Reply:
Let's leave it as is for now